netlify/landings/prism.yaml

```
@@ -0,0 +1,79 @@
+path: /prism
+title: Open Source Mock Server
+color: purple-darker
+relatedTags:
+  - blog-mocking
+hero:
+  subtitle: Build APIs and accelerate API development with instant mock servers
+  title: Write the spec. Prism will do the rest.
```

```suggestion
  title: Free, Realistic, HTTP Mocking for API Descriptions
```

Spectral website is talking about API Description Documents in line with a bunch of API peoples decision to start talking about API Description Documents and API Description Formats instead of API Specifications.